### PR TITLE
air-quality-refactor

### PR DESCRIPTION
**Pull request by kghamilton89**
# 5486
* Refactoring of `airnow` parser configuration explanation.
* Screenshots reflect current UI.
* Description of configuration included.
* Refactored to match format of `basic.md`

### CODE REVIEW

**Comment by kghamilton89** [comment]
@raipc Looks like this build is errored, please check it out Timelord.

**Comment by kghamilton89** [comment]
Long live Timelord!

File: parsers/csv/examples/air-quality.md [hunk 1]
```diff
@@ -35,23 +35,99 @@
...
 
-## Parser Configuration Description
+This document describes the configuration of a CSV Parser to read the CSV file above and insert the data into ATSD. To create a new CSV parser, navigate to the **Data > CSV Parsers** page.
 
-| Field | Setting | Reason |
+![](./images/parsers-page.png)
+
+### Syntax
+
+* Define columns with **Header** because the underlying document lacks a header row.
```
Comment (rodionos) [inline parsers/csv/examples/air-quality.md]: Use `file` instead of document hereandafter.

File: parsers/csv/examples/air-quality.md [hunk 2]
```diff
@@ -35,23 +35,99 @@
...
+* Specify entity with **Default Entity** because there is no entity column.
+* Exclude unneeded data, `Longitude` and `Latitude` positions are constant and unneeded.
+* Parses multiple metrics from one row, `Parameter` column contains the names of several metrics which represent the concentration of various particles.
```
Comment (rodionos) [inline parsers/csv/examples/air-quality.md]: Parses multiple - this sentence doesn't read right.

File: parsers/csv/examples/air-quality.md [hunk 3]
```diff
@@ -35,23 +35,99 @@
...
+
+### Configuration
+
+For information on all settings on the **CSV Parser Editor** page, including those not used in this tutorial, refer to the [**Uploading CSV Files**](../README.md) tutorial.
+
+Enable the parser, define a unique identifier, specify the output of the parser, and indicate the delimiter symbol.
```
Comment (rodionos) [inline parsers/csv/examples/air-quality.md]: The output of parser - what do you mean? Is it format or type of command?

File: parsers/csv/examples/air-quality.md [hunk 4]
```diff
@@ -35,23 +35,99 @@
...
-|  Filter  |  `number('Concentration') >= 0`  |  Used to import data points that contain actual values that are greater than 0 (not empty).  |
-|  Ignored Columns  |  `Latitude`<br>`Longitude`<br>`Unit`  |  Columns that are skipped.<br>For example: Latitude, Longitude, and Unit do not add any value to the time series.<br>Can be added as metric tags using the API or web interface.  |
-|  Header  |  `"Latitude"`<br>`"Longitude"`<br>`"Time"`<br>`"Parameter"`<br>`"Concentration"`<br>`"Unit"`  |  Since the source CSV file does not have column headers, they are assigned and then referenced in the configuration.  |
+Filter | `number('Concentration') > 0` | Samples which have a value less than or equal to `0` are discarded.<br>For more information about `Filter` expressions, refer to [Rule Engine Documentation](https://axibase.com/docs/atsd/rule-engine/filters.html#filter-expression).
+Ignored Columns | See above screenshot. | Define columns which are not included in resultant series commands.
+
+## Commands
+
+Series commands produced by the above parser configuration are shown below.
```
Comment (rodionos) [inline parsers/csv/examples/air-quality.md]: I would also expand with an example of what commands are produced by a sample row?

File: parsers/csv/examples/air-quality.md (hunk 1 above)
Comment (kghamilton89) [reply parsers/csv/examples/air-quality.md]: Understood.

File: parsers/csv/examples/air-quality.md (hunk 2 above)
Comment (kghamilton89) [reply parsers/csv/examples/air-quality.md]: Changed to "extracts multiple metrics"

File: parsers/csv/examples/air-quality.md (hunk 3 above)
Comment (kghamilton89) [reply parsers/csv/examples/air-quality.md]: I was trying to say Command Type.

File: parsers/csv/examples/air-quality.md (hunk 4 above)
Comment (kghamilton89) [reply parsers/csv/examples/air-quality.md]: I don't understand.

**Comment by kghamilton89** [comment]
Final requested change implemented.

**Pull request merged**